rllib/tests/run_regression_tests.py

```diff
@@ -111,8 +111,8 @@
                     trials = run_experiments(
                         experiments, resume=False, verbose=2)
                 finally:
-                    ray.shutdown()
                     _register_all()
+                    ray.shutdown()
```

Comment:
Leave a comment on the reason to reorder so people don't reorder back?

Reply:
I believe the previous is just broken. _register_all actually use internal kv so it should happen before shutdown.
I feel it's just copied around by people actually. SInce there are so many things like this, do you think it's good to add for every line?

Reply:
Putting details at the first instance is enough, if there is such a dependency for ordering.